staging/src/k8s.io/kubectl/pkg/util/openapi/openapi_test.go

```diff
@@ -30,34 +30,6 @@ import (
 
 var fakeSchema = testing.Fake{Path: filepath.Join("..", "..", "..", "testdata", "openapi", "swagger.json")}
 
-var _ = Describe("Reading apps/v1beta1/Deployment from openAPIData", func() {
```

Comment:
should this be replaced with an apps/v1 test?

Reply:
yeah, good catch, fixed.